.github/workflows/new-release.yml

```
@@ -0,0 +1,48 @@
+          - build: mac
+            target: x86_64-apple-darwin
+            os: macos-latest
+            binary: rcr
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v6
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: stable
+      - run: cargo build --locked --release --features memmap2 --target ${{ matrix.platform.target }}
```

actionlint (1.7.10)

property "platform" is not defined in object type {binary: string; build: string; os: string; target: string}

(expression)